crates/uv/tests/tool_list.rs

```diff
@@ -155,9 +166,12 @@ fn tool_list_bad_environment() -> Result<()> {
     success: true
     exit_code: 0
     ----- stdout -----
+    The following tools are already installed:
+
```

Comment:
I don't think we should say this in `uv tool list`, it's implied by the purpose of the command.